TestSwitchApi/TestSwitchApi/Models/Response/CandidateTestStatus.cs

```diff
@@ -0,0 +1,26 @@
+﻿using TestSwitchApi.Models.DataModels;
+
+namespace TestSwitchApi.Models.Response
+{
+    public class CandidateTestStatus
+    {
+        public string TestName { get; set; }
+        public string TestStatus { get; set; }
+
+        private enum Status
+        {
+            NotCompleted,
+            Completed,
+        }
+
+        public CandidateTestStatus()
+        {
+        }
```

Comment:
Doesn't look like we ever use this either? Can we remove it?

Reply:
constructor removed